imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -185,7 +185,10 @@ export function buildOrderSearchRecord(orderId) {
   orderSearch.billingName = shopBilling.address && shopBilling.address.fullName;
   orderSearch.billingPhone = shopBilling.address && shopBilling.address.phone.replace(/\D/g, "");
   orderSearch.shippingName = shopShipping.address && shopShipping.address.fullName;
-  orderSearch.shippingPhone = shopShipping.address && shopShipping.address.phone.replace(/\D/g, "");
+  if (shopShipping.address && shopShipping.address.phone) {
```

Comment:
Did this sneak in from a different PR? Or just fixing an undocumented issue that you came across?

Reply:
I was running into this throwing a `can't find replace of undefined` when trying to place an order. Probably just something that hadn't come up in testing before? Sort of just a workaround but required to keep moving.